source/_components/sensor.wunderground.markdown

```diff
@@ -195,4 +195,8 @@ group:
 Note: While the platform is called “wunderground” the sensors will show up in Home Assistant as “PWS” (eg: sensor.pws_weather).
 </p>
 
+Note that the Weather Underground sensor is added to the entity_registry. Note also that second and subsequent Personal Weather Station ID (pws_id) will have their monitored conditions suffixed with an index number e.g.
```

Comment:
Starting both sentences with `Note` feels repetitive. How about *The second and subsequent...*?

Reply:
Yes but I think it is important to highlight that it appears in entity_registry. How about:

The Weather Underground sensor is added to the entity_registry and any second and subsequent Personal Weather Station ID....

Reply:
Sounds good, it's at least not repetitive :)